src/lib/utils/SynapseClient.ts

```diff
@@ -3306,3 +3306,31 @@ export const changePasswordWithCurrentPassword = (
     BackendDestinationEnum.REPO_ENDPOINT,
   )
 }
+
+// https://rest-docs.synapse.org/rest/POST/user/changePassword.html
+export const changePasswordWithToken = (
+  newPassword: ChangePasswordWithToken,
+) => {
+  return doPost<ChangePasswordWithToken>(
+    '/auth/v1/user/changePassword',
+    newPassword,
+    undefined,
+    undefined,
+    BackendDestinationEnum.REPO_ENDPOINT
+  )
+}
+
+// http://rest-docs.synapse.org/rest/POST/user/password/reset.html
+export const resetPassword = (
```

Comment:
Could you change so the callback endpoint is also a param.  Then Our Sage Account web app would set this to the passwordReset route.  Since it's a page reload after the round trip, we may want to just use a URL parameter instead of the hash, but I'm ok with either.